modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/SpringCodegen.java

```diff
@@ -561,9 +454,9 @@ public void processOpts() {
             importMapping.put("ParameterObject", "org.springdoc.core.annotations.ParameterObject");
         }
 
-        if (useOptional) {
-            writePropertyBack(USE_OPTIONAL, useOptional);
-        }
+//        if (useOptional) {
+//            writePropertyBack(USE_OPTIONAL, useOptional);
+//        }
```

Comment:
@jpfinne overall it looks good. May I know why these 3 lines are commented out?

Reply:
removed